src/Components/Admin/Admin.js

```diff
@@ -30,14 +53,33 @@ class Admin extends React.Component {
    * @param {*} password is the password of the person logging in
    */
   authenticate(username, password) {
-    //write some legit authentication logic in here eventually
-    //if authentication passes then call readFromDatabase(); otherwise not needed
-    this.setState({
-      loggedIn: true,
-      username: username,
-      password: password
-    });
-  }
+    if(username.charAt('@') === -1 || username.charAt('.') === -1)
```

Comment:
What does this do?

Reply:
the .charAt('@') and .charAt('.') part, its not something I've encountered before

Reply:
this was my bootleg version of checking for a valid email. I've updated it so now the form will handle it for us.